docs/components/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -287,14 +307,26 @@ The Connector runtime environment initializes the context and allows the followi
 - Fetch and deserialize the input data as shown in **(1)**. Refer to the [input data](#outbound-connector-input-data) section for details.
 - Execute the Connector's business logic as shown in **(2)**.
 
+###### Basic errors
```

Comment:
```suggestion
#### Basic errors
```

This could also be an H3 if it applies to both inbound/outbound.

Reply:
I'd really prefer to trim this down, as any headers H4 and up tend to actually render smaller than the normal text on a page. I'm planning to handle additional references in https://github.com/camunda/developer-experience/issues/257.